include/modem/lte_lc.h

```diff
@@ -1576,42 +1606,6 @@ int lte_lc_edrx_req(bool enable);
  */
 int lte_lc_edrx_get(struct lte_lc_edrx_cfg *edrx_cfg);
 
-/**
- * Set the RAI value to be used.
- *
- * RAI can be subsequently enabled using lte_lc_rai_req().
- *
- * For reference see 3GPP 24.301 Ch. 9.9.4.25.
- *
- * @deprecated Use @kconfig{CONFIG_LTE_RAI_REQ} and socket option ``SO_RAI`` instead.
```

Comment:
What to use instead was only documented in the code that has been deleted, and will be lost for customers that upgrade from older versions. Please add info about what to use instead in the migration guide for all of the deleted functionality.

Reply:
I've added this statement into the release notes as well. This PR will get closed and I'll create another one to remove deprecated stuff from lte_lc.